spec/models/theme_spec.rb

```diff
@@ -59,6 +59,8 @@
 
   end
 
+  it { is_expected.to belong_to(:remote_theme).dependent(:destroy) }
```

Comment:
Generally we find such tests to be of little value (since they are 1:1 with code) and do not include them.

Reply:
Removed 👍